docs/_guide/properties.md

```diff
@@ -26,77 +26,134 @@ LitElement manages your declared properties and their corresponding attributes.
+
+### Property options
+
+The options object can have the following properties:
+
+<dl>
+<dt>
+
+`attribute`
+
+</dt>
+<dd>
+
+Configure whether the property is associated with an attribute. Default: true. See [Configure observed attributes](#observed-attributes). If `attribute` is false, the `converter`, `reflect` and `type` options are ignored. 
+
+</dd>
+<dt>
+
+`converter`
+
+</dt>
+<dd>
+
+Specifies a [custom converter](#conversion-converter) for converting between properties and attributes. If unspecified, use the [default attribute converter](#conversion-type).
+
+</dd>
+<dt>
+
+`hasChanged`
+
+</dt>
+<dd>
+
+Specify what constitutes a [property change](#haschanged). If unspecified, LitElement uses a 
+strict inequality check (`newValue !== oldValue`) to determine whether the property value has changed.
 
-### Declare properties in a static properties getter
+</dd>
+<dt>
 
-To declare properties in a static `properties` getter:
+`noAccessor`
+
+</dt>
+<dd>
+
+Set to true to avoid generating the default [property accessor](#accessors). Default: false.
+
+</dd>
+<dt>
+
+`reflect`
+
+</dt>
+<dd>
+
+Controls whether property value is reflected back to the associated attribute. Default: false. See  [Configure reflected attributes](#reflected-attributes).
+
+</dd>
+<dt>
+
+`type`
+
+</dt>
+<dd>
+
+Specifies a type hint for converting between properties and attributes. If unspecified, behaves like `type: String`. See [Use LitElement's default attribute converter](#conversion-type). 
+
+</dd>
+
+An empty options object is equivalent to specifying the default value for all options.
+
+<div class="alert alert-info">
+
+**An options object by another name.** This guide uses the descriptive term "options object." In practice the options object is an instance of `PropertyDeclaration`, so you'll see that name if you're using an IDE, or looking at the API reference. By either name, it's an object that defines a set of options.
+
+</div>
+
+### Declare properties in a static properties field
+
+To declare properties in a static `properties` field:
 
 ```js
 static get properties() { 
   return { 
-    prop1: { type: String },
-    prop2: { type: Number },
-    prop3: { type: Boolean }
+    greeting: {type: String},
+    data: {attribute: false},
+    items: {}
   };
 }
 ```
 
-{:.alert .alert-warning}
-<div>
+An empty option object is equivalent to specifying the default value for all options.
 
-**If you implement a static properties getter, initialize your property values in the element constructor.**
+<div class="alert alert-info">
 
-```js
-constructor() {
-  // Always call super() first
-  super();
-  this.prop1 = 'Hello World';
-  ...
-}
-```
-
-Remember to call `super()` first in your constructor, or your element won't render at all.
+If you implement a static properties field, you typically [initialize your property values in the element constructor](#initialize-constructor).
```

Comment:
👍 like the phrasing.

We haven't used the term "reactive properties" in the docs so far, so it would be a fairly significant change across the doc set. (Not saying that's a bad thing, just don't really want to introduce that terminology in this PR.) The term we've used so far is "declared properties."

Reply:
Took this phrasing but s/reactive/declared/ for now.